src/styles/main.scss

```diff
@@ -1,3 +1,17 @@
+@import "./utils.scss/variables";
+@import "./utils.scss/mixins";
+@import "./utils.scss/functions";
+@import "./blocks/calendar.scss";
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
 }
```

Comment:
probably, it's better to create a separate files for normalize, reset and calendar__container